spec/gar_spec.lua

```diff
@@ -2143,12 +2143,6 @@ describe("gar", function()
 	describe("getFundingPlan", function()
 		before_each(function()
 			_G.Balances = {}
-			_G.GatewayRegistry = {}
-		end)
-
-		after_each(function()
-			_G.Balances = {}
-			_G.GatewayRegistry = {}
 		end)
```

Comment:
makes me wonder if global before_each handlers exist

Reply:
fwiw chat gpt says we could workaround by setup the logic in a function and call the setup function in each before_each
<img width="1054" alt="image" src="https://github.com/user-attachments/assets/3d61e5ca-8297-4526-a622-ac653b32bfa5">

Reply:
ours could be like -- `test_helpers.clean_global_state()`. not necessary in the context of this PR just blue sky thinking

Reply:
there is a before_each right above this, this after_each is redundant